public/API/API_GetGameExtended.php

```diff
@@ -50,28 +50,118 @@
  *    string    Expiration                date the claim will expire
  */
 
+use App\Community\Models\AchievementSetClaim;
 use App\Platform\Enums\AchievementFlag;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use Carbon\Carbon;
 
 $gameID = (int) request()->query('i');
 $flag = (int) request()->query('f', (string) AchievementFlag::OfficialCore);
-getGameMetadata($gameID, null, $achData, $gameData, flag: $flag, metrics: true);
 
-if ($gameData === null) {
+$game = Game::with('system')->find($gameID);
+$gameSetClaims = AchievementSetClaim::find($gameID);
+
+// TODO move to seprate file
+if (!function_exists('getParentGameId')) {
+    function getParentGameId(string $title, int $consoleID, int $gameID): ?int
+    {
+
+        $result = Game::where('Title', 'LIKE', '%' . $title . '%')->where('ConsoleID', '=', $consoleID)->first();
+
+        if ($result->Title = $title) {
+            return null;
+        } else {
+            $matchValue = $result->Title;
+            $remainingValue = preg_replace('/(?:^|\s)\[.*Subset - .*\](?:\s|$)/', '', $matchValue);
+            $remainingResult = Game::where('Title', $remainingValue)->first();
+
+            return (int) $remainingResult->ID ? $remainingResult->ID : null;
+        }
+    }
+}
```

Comment:
I feel fairly strongly that this should live somewhere in the `Game` model class.

Reply:
Agreed, I have moved this to a public function in the Game model